docs/Dockerfile-local

```diff
@@ -0,0 +1,19 @@
+FROM python:3.9-buster
```

Comment:
You can inherit `FROM amsterdam/python:3.9-slim-buster` or `FROM amsterdam/python:3.9-buster` - see [here](https://github.com/Amsterdam/docker/tree/master/python) for all images